submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/stats/SubmissionStatsTest.kt

```diff
@@ -341,9 +341,12 @@ class SubmissionStatsTest(
             val stats = webClient.findByTypeAndAccNo("VIEWS", accNo)
             assertThat(stats).isEqualTo(SubmissionStat(accNo, 150L, "VIEWS"))
 
-            val error = assertThrows<WebClientException> { webClient.findByTypeAndAccNo("VIEWS", "STATS-0010") }
+            val notExisting = webClient.findByTypeAndAccNo("VIEWS", "STATS-0010")
+            assertThat(notExisting).isEqualTo(SubmissionStat("STATS-0010", 160L, "VIEWS"))
```

Comment:
@jhoanmanuelms I disable existence check of stats, if submission does not exists stats will be still there, I do not see how this affect system and existence check make the whole process super slow